compat/test/browser/portals.test.js

```diff
@@ -73,7 +86,7 @@ describe('Portal', () => {
 		}
 
 		render(<App />, root);
-		expect(dialog.childNodes.length).to.equal(1);
+		expect(dialog.childNodes.length).to.equal(2);
```

Comment:
I don't know if this is acceptable or not....

Reply:
Not sure if I understand it correctly: Does this mean that we leave a DOM node behind where the `Portal` was?

Reply:
We push an empty textNode (https://github.com/developit/preact/pull/1629/files#diff-efd159ddcc099e90029db3b306b8646aR75) to ensure we are inserting the portal contents and not overriding all the already rendered children.